plugins/modules/network_settings_workflow_manager.py

```diff
@@ -2469,33 +3440,106 @@ def update_network(self, network_management):
 
             net_params = copy.deepcopy(self.want.get("wantNetwork")[network_management_index])
             net_params.update({"site_id": self.have.get("network")[network_management_index].get("site_id")})
-            try:
-                response = self.dnac._exec(
-                    family="network_settings",
-                    function='update_network_v2',
-                    op_modifies=True,
-                    params=net_params,
-                )
-            except Exception as msg:
-                self.msg = (
-                    "Exception occurred while updating the network settings of '{site_name}': {msg}"
-                    .format(site_name=site_name, msg=msg)
-                )
-                self.log(str(msg), "ERROR")
-                self.status = "failed"
-                return self
+            self.log(net_params)
```

Comment:
whats the log level? Log a message and print the values

Reply:
Added log message and log level as Debug.